### PR TITLE
install MNE via pip

### LINKED ISSUE #17: Error when loading PyMNE
Not error were reported when I added and precompile `PyMNE`.
But when I tried to use it, I got the following error:
```
julia> using PyMNE

signal (11): Segmentation fault: 11
in expression starting at REPL[3]:1
_ZN4llvm17PMTopLevelManager16addImmutablePassEPNS_13ImmutablePassE at /Users/likanzhan/.julia/environments/v1.8/.CondaPkg/env/lib/libLLVM-11.dylib (unknown line)
Allocations: 5952326 (Pool: 5949181; Big: 3145); GC: 6

signal (11): Segmentation fault: 11
in expression starting at REPL[3]:1
_PyUnicode_FromId at /Users/likanzhan/.julia/environments/v1.8/.CondaPkg/env/lib/libpython3.10.dylib (unknown line)
wait_for_thread_shutdown at /Users/likanzhan/.julia/environments/v1.8/.CondaPkg/env/lib/libpython3.10.dylib (unknown line)
Py_FinalizeEx at /Users/likanzhan/.julia/environments/v1.8/.CondaPkg/env/lib/libpython3.10.dylib (unknown line)
Py_FinalizeEx at /Users/likanzhan/.julia/packages/PythonCall/dsECZ/src/cpython/pointers.jl:299 [inlined]
#36 at /Users/likanzhan/.julia/packages/PythonCall/dsECZ/src/cpython/context.jl:192
unknown function (ip: 0x14ea6432f)
ijl_apply_generic at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
_atexit at ./initdefs.jl:372
jfptr__atexit_29809 at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/sys.dylib (unknown line)
ijl_apply_generic at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
ijl_atexit_hook at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
ijl_exit at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
jl_exit_thread0_cb at /Applications/Julia-1.8.app/Contents/Resources/julia/lib/julia/libjulia-internal.1.8.dylib (unknown line)
Allocations: 5952326 (Pool: 5949181; Big: 3145); GC: 6

```

And here is the version information of my Julia:

```
Julia Version 1.8.5
Commit 17cfb8e65ea (2023-01-08 06:45 UTC)
Platform Info:
  OS: macOS (arm64-apple-darwin21.5.0)
  CPU: 10 × Apple M1 Max
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, apple-m1)
  Threads: 4 on 8 virtual cores
Environment:
  JULIA_NUM_THREADS = 4
  JULIA_PKG_USE_CLI_GIT = true
```

### PR DESCRIPTION
**Pull request by palday**
fixes #17

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -12,7 +12,8 @@ using Test
     # this is fine because it shows usage and tests out a bit more code
     @test pyconvert(Bool, info.ch_names == info["ch_names"])
     # if this ever works after a compat bump then we know we need to change things
-    @test_broken info.ch_names == info["ch_names"]
+    # can't use `naive @test_broken` in julia 1.10+ because it now requires a bool
+    @test_broken (info.ch_names == info["ch_names"]) isa Bool
```
Comment (kimlaberinto) [inline test/runtests.jl:16]: Nit-picky, but could we move this to a different PR as it doesn't pertain to installing MNE via pip?

**Review by kimlaberinto — COMMENTED** [review]
Changes look good to me and is confirmed to work via: https://github.com/beacon-biosignals/PyMNE.jl/issues/17#issuecomment-1668180367. 

One point regarding other changes being moved to another PR if possible. Otherwise 👍

File: test/runtests.jl (hunk 1 above)
Comment (palday) [reply test/runtests.jl:16]: this was mostly done here because without CI was broken to even do the main change, but sure :)

Comment (kimlaberinto) [reply test/runtests.jl:16]: Oh I see 👍

**Review by kimlaberinto — APPROVED** [review]

**Comment by kimlaberinto** [comment]
Whoops, I thought I approved it already

**Pull request merged**